test/support/factory.ex

```diff
@@ -16,7 +16,7 @@ defmodule Orbit.Factory do
       middle_initial: "A",
       email: sequence(:employee_email, &"fake#{&1}@test.com"),
       badge_number: sequence("employee_badge"),
-      badge_serials: [build(:badge_serial)],
```

Comment:
**question:** what's the purpose of removing this from the factory? We should still have the badge serials available on the back-end, it's just that the controller won't preload them in its query and won't include a `badge_serials` key in its response.

Reply:
No serials seemed like a simpler default value. Usually if you care that this field exists, you should set it in the test, so that changing the factory values doesn't break tests. There was only one test that depended on this and I ended up editing it anyway.